guides/v2.1/frontend-dev-guide/translations/xlate.md

```diff
@@ -21,69 +21,71 @@ A *translation dictionary* is a comma-separated value (.csv) file with at least
 "Address %1 of %2","Adresse %1 von %2"
 ```
 
+*UI text strings* are the text elements in the Magento Admin, including field names, instructions, notifications, and table and grid labels. When localizing Magento, you use translations of these strings using the translation dictionary content.
+
 A *language package* is basically a collection of translation dictionaries for a particular language together with meta-information. You can also distribute language packages to other merchants if you wish. [More information about language packages](#m2devgde-xlate-languagepack).
 
 {:.bs-callout .bs-callout-info}
 To create a language package, the `.csv` file requires additional columns that specify the themes or modules in which the translations were found. For more information, see [Generate a translation dictionary].
 
-### Benefits
+## Benefits of Translations
 
-Benefits include:
+Localizing Magento storefronts and the Admin panel gives your company global presence for support and sales.
 
-*	Two types of language packages:
+*	Magento supports two types of language packages:
 
-	*	Module and theme packages. Included in the `i18n` directory of a module or theme, the Magento application auto-discovers these packages.
-	*	An entire dictionary in one directory.
+	*	Translated Module and theme packages.
 
-		This is intended to be used and distributed as a standalone component (similar to modules and themes).
+		Magento auto-discovers packages included in the `i18n` directory of a module or theme. When installing themes and extensions, consider checking for multiple language versions to download and use.
 
-*	You can use the ready-to-use language packages prepared by other users or you can create your own.
-*	You can create localizations based on existing, or parent, translations using [language inheritance]. Inheritance means (among other things) that if you missed or omitted localizing some phrases or terms, parent translations are used.
-*	You can customize your translations even further by creating more than one version of a translation for the same language.
-
-Depending on your needs, you can use the existing [language packages](#m2devgde-xlate-languagepack) or [translate Magento by yourself](#m2devgde-xlate-translating).
-
-### Changes in Magento 2
+	*	An entire dictionary in one directory.
 
-Changes made:
+		This is intended to be used and distributed as a standalone component (similar to modules and themes).
```

Comment:
This line bugs me a little. It's so...legal-ish. Is it introducing the list?

Reply:
Tweaked!